mail/dovecot2/Portfile

```diff
@@ -22,12 +22,26 @@ long_description    Dovecot is an IMAP and POP3 server for Linux/UNIX-like \
                     it is written in C, it uses several coding techniques to \
                     avoid most of the common pitfalls.
 
+# At present, dovecot2 fails to compile under (at least) 10.6.8 because of
+# the dependency on pandoc -> stack
+# 
+# Now, the dovecot2 documentation
+#     https://wiki2.dovecot.org/CompilingSource
+# states :
+#     pandoc (not strictly required - you can avoid it by using: PANDOC=false ./configure)
+
+set check.os.major  13
+
 depends_build-append \
                     port:gettext \
-                    port:pandoc \
                     port:pkgconfig \
                     port:wget
 
+if {${check.os.major} <= ${os.major}} {
```

Comment:
In all the other ports, we have standardized this verbage to be like this:
```
if {${os.platform} eq "darwin" && ${os.version} > 13} {
```
and then make the version test what you need. Not quibbling about the way you did it, but it's just done that way in every other port, so that's what people can quickly see and understand.

Reply:
I was working from the example on this page : 
     https://trac.macports.org/wiki/PortfileRecipes 
so it might be an idea to update that recipe

Reply:
Quite right -- MacPorts is pushing 15 years old now, and some of it's documentation is clearly out-of-date. The best source I find is to look in Portfiles that do what you want, especially Portfiles that have been updated in the past year or two. Also, there are **so many** PortGroups, and these are **really poorly** documented. You can really only see what they do by looking at the source code of the PortGroup, in _resources/port1.0/group , and see what is going on in there. Even then ...